backup/cronbackup.go

```diff
@@ -117,8 +118,32 @@ func (j *CronJob) backupGC() {
 	}
 }
 
-func (j *CronJob) createBackup() (*apiv1alpha1.Backup, error) {
-	backupName := fmt.Sprintf("%s-auto-%s", j.ClusterName, time.Now().Format("2006-01-02t15-04-05"))
+func (j *CronJob) createBackups() error {
+	// Cannot use uppcase, because RFC 1123 subdomain must consist of lower case alphanumeric characters
+	const nfstype string = "nfs"
+	const s3type string = "s3"
+	if j.BothS3NFS { // create both.
+		if err := j.createOneBackup(nfstype, j.NFSServerAddress); err != nil {
+			return err
+		}
+		if err := j.createOneBackup(s3type, ""); err != nil {
+			return err
+		}
+	} else {
+		if len(j.NFSServerAddress) > 0 {
+			if err := j.createOneBackup(nfstype, j.NFSServerAddress); err != nil {
+				return err
+			}
+		} else {
```

Comment:
Starting two backups at the same time will seriously affect the IO efficiency
It is recommended to support separate backup schedules